File: kotlinamesj/src/main/java/com/nagopy/android/kotlinames/property/required/KRequiredFloatPropertyName.java
package com.nagopy.android.kotlinames.property.required;

import com.nagopy.android.kotlinames.KRequiredPropertyName;
import com.nagopy.android.kotlinames.KSortablePropertyName;

public class KRequiredFloatPropertyName implements KRequiredPropertyName<Float>, KSortablePropertyName<Float> {

    private final String name;

    public KRequiredFloatPropertyName(String name) {
        this.name = name;
    }

    @Override
    public String name() {
        return name;
    }
}

File: kotlinames/src/main/kotlin/com/nagopy/android/kotlinames/property/required/KRequiredFloatPropertyName.kt
package com.nagopy.android.kotlinames.property.required

import com.nagopy.android.kotlinames.KRequiredPropertyName
import com.nagopy.android.kotlinames.KSortablePropertyName

class KRequiredFloatPropertyName(val name: String) : KRequiredPropertyName<Float>, KSortablePropertyName<Float> {

    override fun name(): String = name

}
